pkgs/by-name/el/elfdeps/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchPypi,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "elfdeps";
+  version = "0.2.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-2ZACYS2eCFSOFN2mOWxxRgcc9BUNZ065j0ztDqBSlos=";
+  };
```

Comment:
fetchFromGitHub is preferred unless there is a compelling reason to use Pypi, in that case please add a comment

Reply:
Please explain why it's preferred. It's pypi here because that's what nix-init put here. Nothing in python packaging documentation seems to suggest to use github.